apim/3.x/values.yaml

```diff
@@ -511,7 +511,8 @@ api:
       enabled: false
       prometheus:
         enabled: true
-
+    bridge:
```

Comment:
@kamiiiel Can you please provide the whole bridge configuration block as it's already for the gateway ?